docs/migration.md

```diff
@@ -418,11 +404,86 @@ exporters:
 
 ##### Collection should begin
 
-The OpenTelemetry Collector doesn't have a substitute for this Installed Collector option.
-It supports two options, starting at the beginning or end of a file.
-Starting at the `beginning` will read the entire file every time it's started.
-Starting at the `end` will read only logs appended to file after it's started.
-This is configurable with the `start_at` option.
+The OpenTelemetry Collector substitution for this Installed Collector option is complex,
+as the following steps needs to be performed:
+
+- reading logs using [Filelog Receiver][filelogreceiver]
+- parsing timestamp from the logs using [Logs Transform Processor][logstransformprocessor]
+- marking logs to be removed using [Transform Processor][transformprocessor] processor
+- remove logs using [Filter Processor][filterprocessor]
+
+Let's consider the following example. We want to get logs from `tmp/logs.log` which are at least from 2023 year:
+
+- `tmp/logs.log`
+
+  ```text
+  2020-04-01 10:12:14 Log from 2020
+  2021-01-02 12:13:54 Log from 2021
+  2022-03-07 11:15:29 Log from 2022
+  2023-01-02 10:37:12 Log from 2023
+  ```
+
+- `config.yaml` (only essential parts)
+
+  ```yaml
+  receivers:
+    filelog/log source:
+      include:
+      - tmp/logs.log
+      # - ...
+      ## Adds file path log.file.path attribute, which will be used further in pipeline
+      include_file_path: true
+      ## We would like to read from beginning, as we can choose only between end and beginning
+      start_at: beginning
+      # ...
+  processors:
+    ## The following processor parses out the timestamp from `tmp/logs.log`
+    logstransform/extract timestamp:
+      operators:
+      - type: regex_parser
+        ## Applies only to tmp/logs.log file
+        if: 'attributes["log.file.path"] == "tmp/logs.log"'
+        ## Extracts timestamp to timestamp_field using regex parser
+        regex: '^(?P<timestamp_field>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}).*'
+        timestamp:
+          parse_from: attributes.timestamp_field
+          layout: '%Y-%m-%d %H:%M:%S'
+      ## cleanup timestamp_field
+      - type: remove
+        field: attributes.timestamp_field
+    ## Marks to remove logs older than Sat Dec 31 2022 23:00:00 GMT+0000
+    transform/mark older:
+      log_statements:
+      - context: log
+        statements:
+        # 1672527600000000000 ns is equal to Sat Dec 31 2022 23:00:00 GMT+0000
+        - set(attributes["to_remove"], "true") where time_unix_nano < 1672527600000000000
+    ## Remove logs which has been marked to be removed
+    filter/remove older:
+      logs:
+        exclude:
+          match_type: strict
```

Comment:
You can use ottl directly in filterprocessor now: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/README.md#ottl